sctp/src/queue/pending_queue.rs

```diff
@@ -1,19 +1,21 @@
 use crate::chunk::chunk_payload_data::ChunkPayloadData;
 
-use std::collections::VecDeque;
-use std::sync::atomic::{AtomicBool, AtomicUsize, Ordering};
-use tokio::sync::Mutex;
+use std::{
```

Comment:
We should use the locks from `util::sync` here. Might be best to add a TODO for now